tests/unit/s2n_asn1_time_test.c

```diff
@@ -128,7 +128,7 @@ int main(int argc, char **argv)
      * Also note, the moment timezones come into play, so does daylight savings time. So there are two tests here, across the timezone boundaries.*/
     {
         char *tz = getenv("TZ");
-        setenv("TZ", "US/Pacific", 1);
```

Comment:
Why this change? --> putenv takes a single argument, setenv is correct here.

Reply:
I am surprised that I didnt get a compiler error :O

https://man7.org/linux/man-pages/man3/setenv.3.html
> This function makes copies of the strings pointed to by name and value (by contrast with [putenv(3)](https://man7.org/linux/man-pages/man3/putenv.3.html)).